test/regressions/index.js

```diff
@@ -4,8 +4,8 @@ import * as ReactDOMClient from 'react-dom/client';
 import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
 import webfontloader from 'webfontloader';
 import TestViewer from './TestViewer';
+import 'docs/src/styles.css';
 import 'docs/src/demo-theme.css';
```

Comment:
We don't need this if we are loading `docs/src/styles.css`
`demo-theme.css` is just for presenting the theme.css tab in the demos